src/Component/Hero/index.js

```diff
@@ -1,35 +1,35 @@
 import * as React from 'react';
-import { styled } from '@mui/material/styles';
 import Box from '@mui/material/Box';
-import Paper from '@mui/material/Paper';
 import Grid from '@mui/material/Grid';
 import Image from './../HeroImage';
-//import IconsHomePage from './../IconsHomePage';
+import IconsHomePage from './../IconsHomePage';
 import MainButton from './../MainButton';
 import Typography from '@mui/material/Typography';
-const Item = styled(Paper)(({ theme }) => ({
-    backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
-    ...theme.typography.body2,
-    padding: theme.spacing(1),
-    textAlign: 'center',
-    color: theme.palette.text.secondary,
-}));
+import { createTheme } from "@mui/material/styles";
 
-export default function ColumnsGrid() {
+const theme = createTheme({
+    GridTheme: {
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "space-around",
+        height: 500,
+    }
+});
```

Comment:
Regarding this code, can we use a styled component for this, as in the example below:
![image](https://github.com/MohammadGhawadrah/Stayction/assets/132939680/517d410b-314d-404a-a5a5-6b0eb5eb0a47)